tests/testthat/test-renderer-sixplot.R

```diff
@@ -0,0 +1,76 @@
+test_that("check rowspan, all plots in single row", {
+  plot_of_3_dots_data <- data.frame(x = c(1, 2, 3), y = c(1, 4, 9))
+  plot_of_2_dots_data <- data.frame(x = c(1, 2), y = c(1, 4))
+  plot_of_1_dot_data  <- data.frame(x = c(2), y = c(2))
+  plot_of_4_dots_data <- data.frame(x = c(1, 2, 3, 6), y = c(1, 4, 9, 13))
+  plot_of_5_dots_data <- data.frame(x = c(1, 2, 3, 6, 9), y = c(1, 4, 9, 13, 16))
+  plot_of_6_dots_data <- data.frame(x = c(1, 2, 3, 6, 9, 11), y = c(1, 4, 9, 13, 16, 18))
+
+  plot_of_3_dots <- ggplot(plot_of_3_dots_data, aes(x, y)) +
+    geom_point(size = 2) +
+    ggtitle("Plot of 3 Dots") +
+    xlab("X Axis") + ylab("Y Axis") +
+    theme_animint(rowspan = 2)
+
+  plot_of_2_dots <- ggplot(plot_of_2_dots_data, aes(x, y)) +
+    geom_point(size = 2) +
+    ggtitle("Plot of 2 Dots") +
+    xlab("X Axis") + ylab("Y Axis")
+
+  plot_of_1_dot <- ggplot(plot_of_1_dot_data, aes(x, y)) +
+    geom_point(size = 2) +
+    ggtitle("Plot of 1 Dot") +
+    xlab("X Axis") + ylab("Y Axis") +
+    theme_animint(rowspan = 2)
+
+  plot_of_4_dots <- ggplot(plot_of_4_dots_data, aes(x, y)) +
+    geom_point(size = 2) +
+    ggtitle("Plot of 4 Dots") +
+    xlab("X Axis") + ylab("Y Axis")
+
+  plot_of_4_dots_copy <- ggplot(plot_of_4_dots_data, aes(x, y)) +
+    geom_point(size = 2) +
+    ggtitle("Plot of 4 Dots Copy") +
+    xlab("X Axis") + ylab("Y Axis")
+
+  plot_of_5_dots <- ggplot(plot_of_5_dots_data, aes(x, y)) +
+    geom_point(size = 2) +
+    ggtitle("Plot of 5 Dots") +
+    xlab("X Axis") + ylab("Y Axis")
+
+  plot_of_6_dots <- ggplot(plot_of_6_dots_data, aes(x, y)) +
+    geom_point(size = 2) +
+    ggtitle("Plot of 6 Dots") +
+    xlab("X Axis") + ylab("Y Axis") +
+    theme_animint(row = 1, col = 3)
+
+  plot_list <- list(
+    plot1 = plot_of_1_dot,
+    plot2 = plot_of_2_dots,
```

Comment:
please avoid declaring variables above and then using them only once here.

bad:
```r
variable <- something
plot_list <- list(plot1=variable)
```
good:
```r
plot_list <- list(plot=something)
```